src/coreclr/jit/fgopt.cpp

```diff
@@ -1330,6 +1328,59 @@ BlockSet_ValRet_T Compiler::fgGetDominatorSet(BasicBlock* block)
     return domSet;
 }
 
+//-------------------------------------------------------------
+// fgComputeDominators: Compute new dominators
+//
+// Returns:
+//    Suitable phase status.
+//
+PhaseStatus Compiler::fgComputeDominators()
```

Comment:
It's a bit confusing that we now have `fgComputeDoms` and `fgComputeDominators`, but the confusion should only be temporary...